### PR TITLE
FUN-992 - Add minimumEstimateGasPrice to Functions Coordinator config

### PR DESCRIPTION
**Pull request by justinkaseman**

### CODE REVIEW

File: contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol [hunk 1]
```diff
@@ -39,7 +39,8 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 requestTimeoutSeconds; //                 ║ How many seconds it takes before we consider a request to be timed out
     uint72 donFee; //                                ║ Additional flat fee (in Juels of LINK) that will be split between Node Operators. Max value is 2^80 - 1 == 1.2m LINK.
     uint16 maxSupportedRequestDataVersion; // ═══════╝ The highest support request data version supported by the node. All lower versions should also be supported.
-    uint224 fallbackNativePerUnitLink; // ═══════════╸ fallback NATIVE CURRENCY / LINK conversion rate if the data feed is stale
+    uint224 fallbackNativePerUnitLink; // ═══════════╗ Fallback NATIVE CURRENCY / LINK conversion rate if the data feed is stale
+    uint32 minimumEstimateGasPrice; // ══════════════╝ The lowest amount of wei that will be used as the tx.gasprice when estimating the cost to fulfill the request
```
Comment (bolekk) [inline contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol:43]: This type is too small, only goes to ~4 gwei.

**Review by bolekk — CHANGES_REQUESTED** [review]

File: contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol (hunk 1 above)
Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol:43]: I think we could get away with uint40. Or go up to uint64? May be overkill.

Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol:43]: Changed to uint40 because it packs nicely. This let's us get ~`1,099`gwei

Comment (bolekk) [reply contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol]: I think that's enough.
But does it pack nicely? 224+40 = 264

The PR looks good, I'll take another look tomorrow before approving.

Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol]: >  But does it pack nicely? 224+40 = 264

I moved one of the uint32s down, so it's now:
slot1: 32+32+32+32+72+16+40 = 256
slot2: 224+32 = 256

**Review by KuphJr — APPROVED** [review]
Other than Bolek's comment, it LGTM! Thanks for the fast turnaround.

**Review by bolekk — DISMISSED** [review]

File: contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol [hunk 2]
```diff
@@ -36,10 +36,11 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 feedStalenessSeconds; //                  ║ How long before we consider the feed price to be stale and fallback to fallbackNativePerUnitLink.
     uint32 gasOverheadBeforeCallback; //             ║ Represents the average gas execution cost before the fulfillment callback. This amount is always billed for every request.
     uint32 gasOverheadAfterCallback; //              ║ Represents the average gas execution cost after the fulfillment callback. This amount is always billed for every request.
-    uint32 requestTimeoutSeconds; //                 ║ How many seconds it takes before we consider a request to be timed out
     uint72 donFee; //                                ║ Additional flat fee (in Juels of LINK) that will be split between Node Operators. Max value is 2^80 - 1 == 1.2m LINK.
+    uint40 minimumEstimateGasPrice; //               ║ The lowest amount of wei that will be used as the tx.gasprice when estimating the cost to fulfill the request
```
[the remark this answers is not in the document]
Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol:40]: That would make it more clear and goes in line with our incoming solidity style guideline updates! Changing

File: contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol [hunk 3]
```diff
@@ -36,10 +36,11 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 feedStalenessSeconds; //                  ║ How long before we consider the feed price to be stale and fallback to fallbackNativePerUnitLink.
     uint32 gasOverheadBeforeCallback; //             ║ Represents the average gas execution cost before the fulfillment callback. This amount is always billed for every request.
     uint32 gasOverheadAfterCallback; //              ║ Represents the average gas execution cost after the fulfillment callback. This amount is always billed for every request.
-    uint32 requestTimeoutSeconds; //                 ║ How many seconds it takes before we consider a request to be timed out
     uint72 donFee; //                                ║ Additional flat fee (in Juels of LINK) that will be split between Node Operators. Max value is 2^80 - 1 == 1.2m LINK.
+    uint40 minimumEstimateGasPriceWei; //               ║ The lowest amount of wei that will be used as the tx.gasprice when estimating the cost to fulfill the request
```
Comment (bolekk) [inline contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol:40]: re-align comment

**Review by bolekk — DISMISSED** [review]

**Review by bolekk — APPROVED** [review]

**Pull request merged**